models/pod.go

```diff
@@ -112,6 +116,10 @@ func (pod *Pod) Parse(p *core_v1.Pod) {
 	_, pod.VersionLabel = p.Labels[conf.IstioLabels.VersionLabelName]
 }
 
+func isIstioProxy(pod *core_v1.Pod, container *core_v1.Container, conf *config.Config) bool {
+	return pod.Namespace == conf.IstioNamespace && (pod.Name == "istio-ingressgateway" || pod.Name == "istio-egressgateway" || container.Name == "istio-proxy")
```

Comment:
Note that pod would never be a "istio-ingressgateway" but it should be typically a "istio-ingressgateway-79df7c789f-gtw5j"

Reply:
good point :-)  That should be a "startsWith" logic.  Or, we could limit this to the container.Name test.

Reply:
Fixed